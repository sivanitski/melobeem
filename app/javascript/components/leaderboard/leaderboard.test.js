import Enzyme, { mount, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";

import Leaderboard from "./leaderboard";

Enzyme.configure({
  adapter: new Adapter(),
});

describe("Leaderboard", () => {
  it("Should display loading when state is undefined", () => {
    const inputControl = shallow(<Leaderboard />);

    expect(inputControl.find("div").text()).toEqual("Loading...");
  });
});
