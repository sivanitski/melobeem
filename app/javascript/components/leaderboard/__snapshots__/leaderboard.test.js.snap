// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leaderboard should render Competitors component 1`] = `
<div>
  Loading...
</div>
`;
